src/content/docs/en/tutorial/6-islands/3.mdx

```diff
@@ -74,3 +67,31 @@ Congratulations on completing the Astro blog tutorial! Share your achievement wi
   <Button link='https://twitter.com/intent/tweet?text=Just%20finished%20learning%20how%20to%20build%20my%20first%20Astro%20blog!%20Check%20it%20out%20at%20https://docs.astro.build/%0Avia%20%40astrodotbuild'>Share on Twitter</Button>
   <Button link='https://www.reddit.com/submit?url=https://docs.astro.build/&title=Just%20finished%20learning%20how%20to%20build%20my%20first%20Astro%20blog!'>Share on Reddit</Button>
 </div>
+
+## Next Steps
+
+You can enhance this project's final code with one of our tutorial extensions, or start your next Astro project!
+
+
+<CardGrid>
+  <LinkCard
+    title="Extend with content collections"
+    description="Refactor the tutorial code to manage your blog posts with content collections."
+    href="/en/tutorials/add-content-collections/"
+  />
+   <LinkCard
+    title="Extend with view transitions"
+    description="Add view transitions to the tutorial code to customize page navigation."
+    href="/en/tutorials/add-view-transitions/"
+  />
+  <LinkCard
+    title="Start a new Astro Project"
+    description="Begin a new empty project, or use an existing Astro theme template."
+    href="/en/install-and-setup/"
```

Comment:
is this "good" to have? I meant if someone already goes through the [blog tutorial](https://docs.astro.build/en/tutorial/1-setup/2/) , they should be fairly aware about `create astro@latest`